config/crd/bases/nfd.openshift.io_nodefeaturediscoveries.yaml

```diff
@@ -35,36 +34,72 @@ spec:
           metadata:
             type: object
           spec:
-            description: NodeFeatureDiscoverySpec defines the desired state of NodeFeatureDiscovery
+            description: Specification of the desired behavior of the Node Feature
+              Discovery
```

Comment:
Is "Discovery" on line 38 supposed to be on previous line 37 ?

Reply:
This is automated by `make manifests` to fit the best practice of 80 column text